readme.md

```diff
@@ -45,3 +45,25 @@ Copyright Quentin Lebastard and Stefan Kerkmann 2021.
 * Splithand detection - ok
 * Bootloader bodge - ok, can enter reliably everytime
 * Matrix - ok
+
+## BOM
+
+| Qty. |                      Part                      |                                                                Link                                                                |
+| ---- | ---------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| 2    | WeAct STM32F411 Blackpill                      | [AliExpress](https://www.aliexpress.com/item/1005001456186625.html)                                                                |
+| 2    | 64Kbit STM M95 SPI EEPROM (SOP-8 package size) | [LCSC Electronics](https://lcsc.com/product-detail/EEPROM_STMicroelectronics-M95640-WMN6TP_C140882.html)                           |
+| 4    | 5.1k 1206 Resistors                            | [LCSC Electronics](https://lcsc.com/product-detail/Chip-Resistor-Surface-Mount_UNI-ROYAL-Uniroyal-Elec-1206W4F5101T5E_C26033.html) |
```

Comment:
I just realized that you need 6 resistors in total not 4... Although one would probably order 100 as they are cheap.